samples/batch-process-document.v1beta3.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright 2020 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const uuid = require('uuid');
+
+async function main(
+  projectId = 'YOUR_PROJECT_ID',
+  location = 'YOUR_PROJECT_LOCATION',
+  processorId = 'YOUR_PROCESSOR_ID', // Create this in the Cloud Console
+  gcsInputUri = 'gs://cloud-samples-data/documentai/invoice.pdf',
+  gcsOutputUri = 'output-bucket',
+  gcsOutputUriPrefix = uuid.v4()
+) {
+  // [START documentai_batch_process_document]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'YOUR_PROJECT_ID';
+  // const location = 'YOUR_PROJECT_LOCATION'; // Format is 'us' or 'eu'
+  // const processorId = 'YOUR_PROCESSOR_ID';
+  // const gcsInputUri = 'YOUR_SOURCE_PDF';
+  // const gcsOutputUri = 'YOUR_STORAGE_BUCKET';
+  // const gcsOutputUriPrefix = 'YOUR_STORAGE_PREFIX';
+
+  // Imports the Google Cloud client library
+  const {
+    DocumentProcessorServiceClient,
+  } = require('@google-cloud/documentai').v1beta3;
+  const {Storage} = require('@google-cloud/storage');
+
+  const clientOptions = {
+    apiEndpoint: 'us-documentai.googleapis.com',
+  };
+
+  const client = new DocumentProcessorServiceClient(clientOptions);
+  const storage = new Storage();
+
+  async function batchProcessDocument() {
+    const name = `projects/${projectId}/locations/${location}/processors/${processorId}`;
+
+    const inputConfig = {
```

Comment:
Personal opinion thing - I find it much easier to read these snippets when we have a single `request` object that inlines all properties I will need at once, as opposed to declaring some things individuals (like `inputConfigs` and `inputConfig`). Curious what others think :) @bcoe @fhinkel (note: this is non-blocking feedback)

Reply:
Hmm, interesting! I can see your point about `inputConfig`--it's arbitrarily outside of the request block, when compared to `outputConfig` (artifact of debugging 😄 ).

I think resource names should have a place of distinction, however, so that their format is clearly shown.

Reply:
Made the changes to `inputConfig`; let me know if you want me to move `parent` as well.